### PR TITLE
feat: Add Emoji flag to the language selector

### PR DESCRIPTION
**Pull request by lipis**
### Preview: https://excalidraw-git-emoji.excalidraw.vercel.app/

-----

- [x] Test on all supported browsers
	- [x] macOS
	- [x] Windows
	- [x] Linux
	- [x] Android
	- [x] iPhone
- [ ] Fallback?! 

Screenshot 2020-12-19 at 11 35 34 PM

---- 

Screenshot 2020-12-19 at 11 35 46 PM

### CODE REVIEW

**Comment by dwelle** [comment]
:+1: 

Win7 doesn't display the flag, but to hell with Win7 (and it also has a textual fallback):

image

**Comment by lipis** [comment]
> Win7 doesn't display the flag, but to hell with Win7 (and it also has a textual fallback):

It's around ~1% of our userbase on Analytics..

File: src/i18n.ts [hunk 1]
```diff
@@ -6,46 +6,49 @@ import percentages from "./locales/percentages.json";
...
+  { lng: "sv-SE", emoji: "🇸🇪", label: "Svenska" },
+  { lng: "tr-TR", emoji: "🇹🇷", label: "Türkçe" },
+  { lng: "uk-UA", emoji: "🇺🇦", label: "Українська" },
+  { lng: "zh-CN", emoji: "🇨🇳", label: "简体中文" },
+  { lng: "zh-TW", emoji: "🇹🇼", label: "繁體中文" },
 ];
 
-export const languages: Language[] = [{ lng: "en", label: "English" }]
+export const languages: Language[] = [
+  { lng: "en", emoji: "🇬🇧", label: "English" },
```
Comment (lipis) [inline src/i18n.ts:50]: or 🇺🇸?

Comment (dwelle) [reply src/i18n.ts:50]: Let's pay respects to the origin and leave it as GB? :)

**Comment by ad1992** [comment]
looks cool ❤️

**Pull request closed without merging**